hack/generated/controllers/suite_test.go

```diff
@@ -31,6 +38,13 @@ const (
 )
 
 var testContext ControllerTestContext
+var envtestContext *EnvtestContext
+
+type EnvtestContext struct {
```

Comment:
Is it `Envtest` or `EnvTest`? I guess `envtest` is the name of the module so it's all one word?